src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/HealthCertificateValidationService.swift

```diff
@@ -109,6 +110,7 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
 	
 	// MARK: - Private
 	
+	private var rulesDownloadService: RulesDownloadServiceProviding?
```

Comment:
Why is the service optional when it it passed in the initalizer?
Why is it private?
Is it set from outside?
Otherwise, you could declare it as let and remove the optionality, so you do not have to think about what error is thrown if the service is nil when trying to trigger a download ;)

Reply:
I removed the optional and made it a constant, but i think it should remain private as long as we don't need to access it from outside right ?

Reply:
Yes absolutely, it should stay `private`